### DISCUSSION TITLE
Add option to disable WiFi in game/after boot

### DISCUSSION THREAD

**Discussion by ourigen** [OnionUI/Onion#1439] [Ideas]
I appreciate that on a MM+ we can sync clock time using WiFi. However, beyond that I don't normally use WiFi and would prefer to turn it off to save battery.

It would be great to have an option in Tweaks to "Disable WiFi in game", or "Disable WiFi after boot" where WiFi is turned on only at boot to check for updates and sync the clock and then turned off automatically until next reboot.

**Comment by XK9274** [comment]
Gotta agree with this (it's also something we've discussed before between ourselves, I think schmurtz suggested the exact same thing a while ago) so it'll likely make the to-do list 👍

Thanks for getting involved!

**Reply by SeriousPat** [reply]
Is this implemented, already?

**Comment by SeriousPat** [comment]
Quick and Dirty Solution:
Add File "disablewlan.sh" in ".tmp_update/startup" directory.
Code in file: ifconfig wlan0 disable

The Device will disable wlan after the timesync. The wlan setting will keep "on" in the json file, so the wlan will be re enalbed after next boot.
